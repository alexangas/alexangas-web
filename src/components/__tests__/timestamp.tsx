import React from "react"
import renderer from "react-test-renderer"

import Timestamp from "../timestamp"

describe(`Timestamp`, () => {
  it(`renders no timestamp`, () => {
    const testRenderer = renderer.create(<Timestamp />)
    const tree = testRenderer.toJSON()
    expect(tree).toMatchSnapshot()
  })

  it(`renders timestamp`, () => {
    const testRenderer = renderer.create(
      <Timestamp dateTime="2020-05-08T12:58" />
    )
    const tree = testRenderer.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
