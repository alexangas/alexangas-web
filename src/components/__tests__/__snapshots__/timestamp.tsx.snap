// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timestamp renders no timestamp 1`] = `null`;

exports[`Timestamp renders timestamp 1`] = `
<div
  className="is-inline-block-tablet"
>
  <small>
    <time
      dateTime="2020-05-08T12:58"
    >
      8 May 2020
    </time>
  </small>
</div>
`;
